pydatalab/pydatalab/models/people.py

```diff
@@ -62,6 +62,27 @@ def add_missing_verification(cls, v):
         return v
 
 
+class DisplayName(str):
+    """
+    A constrained string less than 150 characters long
+    that cannot be empty and should contain only alphabetic characters and spaces.
+    """
+
+    max_length = 150
+
+    def __new__(cls, value):
+        if not value.strip():
+            raise ValueError("Display name cannot be empty.")
+
+        if not all(char.isalpha() or char.isspace() for char in value):
+            raise ValueError("Display name can only contain alphabetic characters and spaces.")
+
+        if len(value) > cls.max_length:
+            raise ValueError(f"Display name must be at most {cls.max_length} characters long.")
+
```

Comment:
I wonder if we should also canonicalize to "title case", e.g, first char of each name is capitalized. Awkward cases I can think of:

double-barelled names, Dutch style `van der Waals` or `t'Hooft`, American names with numbers at the end (John Smith III).

This could be done in the validator but not raise errors, just effects database storage

Probably no need to do that in this PR as I think it will be a bit annoying

Reply:
I think I will retract all my comments about validation beyond checking if the name is empty, I fell down this trap: https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/